src/types.ts

```diff
@@ -1776,3 +1776,20 @@ export type TranslationLanguages =
   | 'vi'
   | 'zh'
   | 'zh-TW';
+
+export type RateLimitsInfo = {
+  limit: number;
+  remaining: number;
+  reset: number;
+};
+
+export type RateLimitsMap = {
+  [key: string]: RateLimitsInfo;
+};
```

Comment:
@ruggi Do we know exactly what key could be? If yes, we can add more strong types:

```ts
export type RateLimitsType = "key1" | "key2" | "key3";
export type RateLimitsMap = Record<RateLimitsKey, RateLimitsInfo>

```

Reply:
it's one key per endpoint, so the problem here is keeping that in sync

Reply:
If it's not going to change frequently (weekly/monthly basis), we can just build types it for once. If you can point me to code in chat repo, I can add it tomorrow :)

Reply:
@nhannah what do you think? Worth it?